core/src/components/toggle/test/label/toggle.e2e.ts

```diff
@@ -169,3 +169,47 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, config, screen
     });
   });
 });
+
+configs({ modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('toggle: start placement'), () => {
+    test('should render the label in the start position', async ({ page }) => {
+      await page.setContent(
+        `
+         <ion-toggle label-placement="start">Label</ion-toggle>
+       `,
+        config
+      );
+
+      const toggle = page.locator('ion-toggle');
+      await expect(toggle).toHaveScreenshot(screenshot(`toggle-label-start-justify-start`));
```

Comment:
```suggestion
      await expect(toggle).toHaveScreenshot(screenshot(`toggle-label-start`));
```
This test isn't checking for justify so we need to update the file name.

Make sure to delete the `ionic-md` files for `toggle-label-start-justify-start` like `toggle-label-start-justify-start-ionic-md-ltr-light-Mobile-Chrome-linux.png`

Reply:
Done!